tests/test_scrump.py

```diff
@@ -378,15 +370,10 @@ def test_scrump_plus_plus_self_join(T_A, T_B, percentages):
 
             np.random.seed(seed)
             ref_P, ref_I = naive.prescrump(T_B, m, T_B, s=s, exclusion_zone=zone)
-            ref_mp = naive.scrump(T_B, m, T_B, percentage, zone, True, s)
-            for i in range(ref_mp.shape[0]):
-                if ref_P[i] < ref_mp[i, 0]:
-                    ref_mp[i, 0] = ref_P[i]
-                    ref_mp[i, 1] = ref_I[i]
-            ref_P = ref_mp[:, 0]
-            ref_I = ref_mp[:, 1]
-            # ref_left_I = ref_mp[:, 2]
-            # ref_right_I = ref_mp[:, 3]
+            ref_P_aux, ref_I_aux, _, _ = naive.scrump(
```

Comment:
are you okay with the names ` ref_P_aux` / `ref_I_aux` ?

Reply:
It's okay. Not the greatest but not the worst